docs/guides/setup/configuring_data_contexts/how_to_instantiate_a_data_context_without_a_yml_file.md

```diff
@@ -31,162 +31,207 @@ Steps
 
     The following example shows a Data Context configuration with an SQLAlchemy datasource and an AWS S3 bucket for all metadata stores, using default prefixes. Note that you can still substitute environment variables as in the YAML based configuration to keep sensitive credentials out of your code.
 
-    ```python
-    from great_expectations.data_context.types.base import DataContextConfig, DatasourceConfig
-    from great_expectations.data_context import BaseDataContext
-
-    data_context_config = DataContextConfig(
-        datasources={
-            "my_sqlalchemy_datasource": DatasourceConfig(
-                class_name="SqlAlchemyDatasource",
-                credentials={
-                    "drivername": "custom_drivername",
-                    "host": "custom_host",
-                    "port": "custom_port",
-                    "username": "${USERNAME_FROM_ENVIRONMENT_VARIABLE}",
-                    "password": "${PASSWORD_FROM_ENVIRONMENT_VARIABLE}",
-                    "database": "custom_database",
+```python
+from great_expectations.data_context.types.base import DataContextConfig, DatasourceConfig, S3StoreBackendDefaults
+
+data_context_config = DataContextConfig(
+    datasources={
+        "sql_warehouse": DatasourceConfig(
+            class_name="Datasource",
+            execution_engine={
+                "class_name": "SqlAlchemyExecutionEngine",
+                "credentials": {
+                    "drivername": "postgresql+psycopg2",
+                    "host": "localhost",
+                    "port": "5432",
+                    "username": "postgres",
+                    "password": "postgres",
+                    "database": "postgres",
                 },
-            )
-        },
-        store_backend_defaults=S3StoreBackendDefaults(default_bucket_name="my_default_bucket"),
+            },
+            data_connectors={
+                "default_runtime_data_connector_name": {
+                    "class_name": "RuntimeDataConnector",
+                    "batch_identifiers": ["default_identifier_name"],
+                },
+                "default_inferred_data_connector_name": {
+                    "class_name": "InferredAssetSqlDataConnector",
+                    "name": "whole_table",
+                },
+            }
+        )
+    },
+    store_backend_defaults=S3StoreBackendDefaults(default_bucket_name="my_default_bucket"),
     )
-    ```
+```
 
```

Comment:
Your code blocks need to be reviewed for formatting.  As they stand now they are causing these non-code sections to be wrapped in a code block in the netlify preview, as well.

Reply:
For reference: These blocks were rendered as code blocks because they were indented in the markdown.  Removing the excess indent reverted them to standard text.